be/src/runtime/mem_tracker.cpp

```diff
@@ -124,14 +124,17 @@ std::shared_ptr<MemTracker> MemTracker::create_tracker_impl(
     std::string reset_label;
     MemTracker* task_parent_tracker = reset_parent->parent_task_mem_tracker();
     if (task_parent_tracker) {
-        reset_label = fmt::format("{}:{}", label, split(task_parent_tracker->label(), ":")[1]);
+        reset_label = fmt::format("{}&{}", label, split(task_parent_tracker->label(), "&")[1]);
```

Comment:
Why change to `&`?

Reply:
Because `:` has many meanings as a separator in task mem tracker label, and `&` is only used to split out `queryId` or `loadID` or `tabletID`.

The latest code replaces `&` with `#`, which seems a little nicer :)